lib/jet/daemon/peers.js

```diff
@@ -64,7 +65,8 @@ exports.Peers = function (jsonrpc, elements) {
       try {
         jsonrpc.dispatch(peer, message)
```

Comment:
Why is this failing though? Can we log "message" when this is failing? One of the things that makes `jsonrpc.dispatch` fail is if the message is not a valid JSON message.

Also, your fix just kind of "patches" things, but doesn't really address the cause of the failure, which makes me worry that the issue that is plaguing the local peer could affect remote peers as well.